.github/workflows/bot.yml

```diff
@@ -83,12 +89,13 @@ jobs:
           java-version: '8'
           distribution: 'adopt'
           architecture: x64
+          cache: maven
       - name: Build Project
         env:
           SCALA_PROFILE: ${{ matrix.scalaProfile }}
           SPARK_PROFILE: ${{ matrix.sparkProfile }}
         run:
-          mvn clean install -D"$SCALA_PROFILE" -D"$SPARK_PROFILE" -DskipTests=true $MVN_ARGS
+          mvn clean install -T 2 -D"$SCALA_PROFILE" -D"$SPARK_PROFILE" -DskipTests=true $MVN_ARGS
```

Comment:
@yihua  @codope @xushiyan   : I remember we removed the multi-threading for some reason. can you folks recollect?

Reply:
can't recall exactly. it was for test stability reason. plus did not see much speed up

Reply:
Some tests (mostly functional or integration tests) cannot run successfully and stably in parallel due to the assumption of sequential execution.

Reply:
The mvn build using multi-threading without running tests may be fine.  We can try it.